src/main/java/frc/robot/BeamBreak.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+
+/** Add your docs here. */
+public class BeamBreak {
```

Comment:
Given that we will be having two (at minimum) beam breaks, I think this class can be made modular as either an abstract class (which intake & other subsystems can use later) with an argument being the DIO channel.